apps/backoffice/src/lib/be/__tests__/institutions.test.ts

```diff
@@ -161,4 +190,97 @@ describe("Institutions", () => {
       expect(getInstitutionById).toHaveBeenCalledWith(mocks.institution.id);
     });
   });
+  describe("retrieveUserGroups", () => {
+    it("should return the groups found for the institution", async () => {
+      const getInstitutionGroups = vi.fn(() =>
+        TE.right(mocks.institutionGroups)
+      );
+      getSelfcareClient.mockReturnValueOnce({
+        getInstitutionGroups
+      });
+
+      const result = await retrieveUserGroups(
+        mocks.institutionGroups.content[0].institutionId
+      );
+
+      expect(getInstitutionGroups).toHaveBeenCalledWith(
+        mocks.institutionGroups.content[0].institutionId,
+        undefined,
+        undefined
+      );
+      expect(result).toEqual({
+        value: mocks.institutionGroups.content.map(userGroupResources => ({
+          id: userGroupResources.id,
+          name: userGroupResources.name
+        })),
+        pagination: {
+          count: mocks.institutionGroups.totalElements,
+          limit: mocks.institutionGroups.size,
+          offset: mocks.institutionGroups.number
+        }
+      });
+    });
+
+    it("should rejects", async () => {
+      const getInstitutionGroups = vi.fn(() => TE.left({ message: "error" }));
+      getSelfcareClient.mockReturnValueOnce({
+        getInstitutionGroups
+      });
+
+      expect(
+        retrieveUserGroups(mocks.institutionGroups.content[0].institutionId)
+      ).rejects.toThrowError();
```

Comment:
`chore:` check the specific error that was raised (i.e.: `ManagedInternalError`)

Reply:
[4ce0326](https://github.com/pagopa/io-services-cms/pull/1007/commits/4ce0326ee02d74d1979064291f7fad04d82ab937)